src/jdbc/cubrid/jdbc/jci/UStatement.java

```diff
@@ -2140,7 +2140,7 @@ private Object readAAttribute(int index, UInputBuffer inBuffer)
 		String charsetName;
 
 		size = inBuffer.readInt();
-		if (size <= 0)
+		if (size < 0)
```

Comment:
size 0 is allowed?

Reply:
This is a bug fix to distinguish '' and null in bit varying type. as a result of my test, there is no effect on other types.